modules/nw-traditional-sharding.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-cluster-traffic-ingress-controller.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-traditional-sharding_{context}"]
+== Traditional sharding example
+
+An Ingress Controller `finops-router` is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `finance` and `ops`:
```

Comment:
Same as the above, better change to:
An Ingress Controller `finops-router` is configured with the label selector "spec.namespaceSelector.matchExpressions" with key name and values in [finance, ops]

Reply:
:heavy_check_mark: